### ISSUE TITLE
Server running on old code

### ISSUE THREAD

**Issue by Ornias1993** [meitar/fetlife-aslsearch#42]
@meitar The google scripts server seems to be running on old code (again), which means it spews out 404 errors again. Please update the master server.

**Comment by lerouxn** [comment]
I think this project has been dropped, never worked since 2014 or something.

**Comment by DRSteven** [comment]
the "slow search" does work again for me (a couple of days ago)
but the fast search, on the separate page doesn't

**Comment by AuraAB** [comment]
Yeah like Ornias said, the server just needs updating for the fast search. If you have a little you can spare, the best way to grab meitar's attention is to donate to him to show your appreciation of his tool, support him and give him some incentive to update it.

**Comment by lerouxn** [comment]
I'd glady drop some btc on it if it worked once in my life. @meitar feel free to ping me when it's done and I'll deliver!

**Comment by meitar** [comment]
> I'd glady drop some btc on it if it worked once in my life. @meitar feel free to ping me when it's done and I'll deliver!

This is the definition of a payment, not a donation. Treating me like a vendor or an employee is the most reliable way to get me to avoid performing the action you'd like me to do.

**Comment by Ornias1993** [comment]
Indeed @meitar this sounds more like gloryfied blackmail.

**Comment by Ornias1993** [comment]
Server seems to be fully offline now.
Working on getting one back online!
